feature/better-brackets

martindevans opened the pull request:
Replaced frame brackets on the outside of the frame with many more printed brackets fit into the inside of the frame. These fit more neatly into the profile and since they're printed we can have one at every corners without having to buy a load more metal brackets. I've removed the metal brackets which we had previously - obviously I'm not sure if we'll need to keep them or not (i.e. will just plastic parts be strong enough) - but we can always add them back later with no trouble since they're entirely outside the internal space of the printer so they won't interfere with anything.

 - Added `XY-Axis.scad` which includes the entire X and Y axes (they're pretty integral to each other, they can be split later if we need to)
 - Renamed `L-Brackets.scad` to a more general `Brackets.scad`
 - Added a todo part into ElectronicsBay.scad (the single plastic part which forms the bottom of the printer/electronics bay
 - Added a todo part into XY-Axis.scad (the single plastic part which forms the top of the printer/XY-frame

martindevans committed:
Replaced frame brackets on the outside of the frame with many more printed brackets fit into the inside of the frame.

 - Added `XY-Axis.scad` which includes the entire X and Y axes (they're pretty integral to each other, they can be split later if we need to)
 - Renamed `L-Brackets.scad` to a more general `Brackets.scad`
 - Added a todo part into ElectronicsBay.scad (the single plastic part which forms the bottom of the printer/electronics bay
 - Added a todo part into XY-Axis.scad (the single plastic part which forms the top of the printer/XY-frame

datkinson commented:
Looks good.

Only comments I have are:

1. The holes closest to the corner may not be able to fit a nut and bolt in both and still be able to tighten them.
2. I'm not sure the brackets wont flex too much even made of PLA.

martindevans commented:
1. Easy fix if it's a problem, don't merge this until you've tested with the ones you're printing
2. I considered a diagonal part across the bracket. That makes it significantly harder to install but gives it a lot more rigidity.

martindevans committed:
Changes driven by testing by Dan.
 - Larger screw holes
 - Reduced number of screws on one half of the bracket to make it possible to fit all the nuts
